eng/compliance/Guardian/BinSkimConfig.xml

```diff
@@ -0,0 +1,26 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) Microsoft Corporation. Use of this source code is governed by a BSD-style license that can be found in the LICENSE file. -->
+<!--
+  To figure out what to put in this file, run this command:
+
+    {nuget-cache}\Microsoft.CodeAnalysis.BinSkim.{version}\tools\netcoreapp3.1\win-x64\BinSkim.exe
+      export-config {target-location}\BinSkimConfigFull.xml
+
+  Then, search in the resulting file to find the XML element with the configuration you want and
+  copy it into this file. For example, searching for "BA3001" (as seen in SDL tool logs) shows how
+  to disable "BA3001.EnablePositionIndependentExecutable.Options".
+-->
+<Properties Type="PropertiesDictionary">
+  <!-- Go doesn't build PIE executables by default: https://github.com/microsoft/go/issues/104 -->
```

Comment:
In case it makes a difference, windows defaults to PIE since go1.19: https://github.com/golang/go/issues/35192

Reply:
Hmm, the commit that closed the issue actually says it's in go1.15, is there something else in go1.19 that ends up enabling it?

But either way, I last touched this file for go1.16 or go1.17, so I'll try a fresh run without this exception and see what happens.

Reply:
Wops, I meant 1.15.

Reply:
BinSkim sees that the Windows binaries are PIE, but it complains about the Linux binaries. (Every arch.) I added a note to https://github.com/microsoft/go/issues/104 and I'll add some more detail to this comment. I think the exception has to stay for now--I haven't been able to find a way to filter a rule for some files and not others.